blocks/header/header.css

```diff
@@ -186,11 +292,41 @@ header nav .nav-sections ul > li > ul > li {
   font-weight: 500;
 }
 
+header nav .nav-tools p a {
+  text-decoration: none;
+  font-family: var(--login-button-font-family);
+  font-size: smaller;
+  padding-left: 10px;
+  padding-right: 10px;
```

Comment:
Padding on the "Log in" link is 24px, making the button larger. I think it's important to adjust it, the difference it's quite big.

Reply:
in feature branch also, I can see padding I've kept as 25px. Anything I'm missing.?

Reply:
Done